release-notes/v2.0.0.md

```diff
@@ -10,6 +10,16 @@
 
 The following table lists the resolved issues for HPE CSI Driver for Kubernetes v2.0.0.
```

Comment:
Can we combine these two know issues into a single table ?

Reply:
One is a CSI driver specific and other is csp specific , so made two different tables as per the last release notes ( We have addressed in two different places)